src/core/font_substitutions.js

```diff
@@ -281,6 +281,26 @@ const substitutionMap = new Map([
       fallback: "Helvetica-BoldOblique",
     },
   ],
+  [
+    "Wingdings",
+    {
+      local: ["Wingdings", "URWDings", "New Dingbats"],
+      style: NORMAL,
+    },
+  ],
+  [
+    "Wingdings-Regular",
+    {
+      alias: "Wingdings",
+    },
+  ],
+  [
+    "Wingdings-Bold",
+    {
+      local: ["Wingdings", "URWDings", "New Dingbats"],
```

Comment:
`alias: "Windings"` should be enough (it will reuse the local from Windings and append a `Bold`).

Reply:
We should probably just alias that to Wingdings, since (my copy of) Windows doesn't actually have a bold version of this font and generally speaking I don't believe that symbol-fonts have e.g. bold/italic versions.